tests/unit/test-util/test.file-filter.js

```diff
@@ -1,15 +1,13 @@
 /* @flow */
-import path from 'path';
-
 import {describe, it} from 'mocha';
 import {assert} from 'chai';
 
-import {FileFilter, normalizeResolve} from '../../../src/util/file-filter';
+import {FileFilter, isSubDir} from '../../../src/util/file-filter';
 
 describe('util/file-filter', () => {
 
   describe('default', () => {
-    const defaultFilter = new FileFilter();
+    const defaultFilter = new FileFilter({sourceDir: '.'});
```

Comment:
In situations where the tests rarely need to change one of the parameters, I think it's helpful to create a helper function:
````diff
diff --git a/tests/unit/test-util/test.file-filter.js b/tests/unit/test-util/test.file-filter.js
index a939c5d..34a5d4f 100644
--- a/tests/unit/test-util/test.file-filter.js
+++ b/tests/unit/test-util/test.file-filter.js
@@ -6,6 +6,11 @@ import {FileFilter, isSubDir} from '../../../src/util/file-filter';
 
 describe('util/file-filter', () => {
 
+  const newFileFilter = (params = {}) => new FileFilter({
+    sourceDir: '.',
+    ...params,
+  });
+
   describe('default', () => {
     const defaultFilter = new FileFilter({sourceDir: '.'});
 
````